python/graphstorm/gconstruct/transform.py

```diff
@@ -164,15 +164,21 @@ def pre_process(self, feats):
             # It will load all data into main memory.
             feats = feats.to_numpy()
 
+        feats[feats == None] = "MAGIC_NONE"
```

Comment:
what does this do?

Reply:
can `None` be the value of the elements in the numpy array?

Reply:
None is a special object. We can not do np.unique on an array with None value.
Here I try to convert None into MAGIC_NONE, so that we can do np.unique.

Reply:
We don't need this. I have updated the implementation.